### ISSUE TITLE
Generate assests on merge using Github actions

### ISSUE THREAD

**Issue by ccordoba12** [ColinDuquesnoy/QDarkStyleSheet#257]
@dpizetta mentioned in https://github.com/ColinDuquesnoy/QDarkStyleSheet/pull/253#issuecomment-761851591:

> As the process of generating those resources is not automatized yet, I would recommend we use the develop branch to PR's, so we can keep the fully functional code in the master (for Python and C++).

I think I could help with that by using Github actions to generate our assests when a pull request is merged against `deveolp` and commit them to the repo.

@dpizetta, what do you think?

**Comment by dpizetta** [comment]
@ccordoba12 of course, essentially we need to run the script process_qrc.  Tests and other ones are ok, see [here](https://travis-ci.org/github/ColinDuquesnoy/QDarkStyleSheet/builds/755757203). 

We are using Tox to run the processes so we can use the same configuration locally on our machines to test.  `tox.ini` has all possible configurations and `.travis.yml` has those to run over the server. 

The build command on tox (`pyqt5-build: python example/example.py --qt_from=pyqt5 --test` ) also generate the screenshoots we see in docs/README.

Some steps we need to generate we need:

[  ] Associate CI/CD to PR (require pass);
[  ] Run and fix build (py38-pyqt5-build) over the server - we had some problems with deps;
[  ] Save resources generated in CI/CD into the repository (update resources);

If you or another one can help me it would be great. Tks
